docs/development/doc_howto.rst

```diff
@@ -157,7 +157,7 @@ please refer to it for more details.
 
 The tooltip is the text that appears when you hover over the thumbnail. It is taken from the first line 
 of the docstring of the tutorial. You can change it by editing the docstring. See e.g.
- `Analysis 1 Tutorial <https://github.com/gammapy/gammapy/blob/main/examples/tutorials/starting/analysis_1.py#L5>`__.
+ `Analysis 1 Tutorial <https://github.com/gammapy/gammapy/blob/4f7f73e493df78ceee59c9884e0ad4750e1fa97c/examples/tutorials/starting/analysis_1.py#L5>`__.
```

Comment:
I am puzzled here. Why would the first one fail? It works for me. The link you propose is that of a specific commit.

Reply:
It was not working in the tests CI... I was puzzled too.